site/src/pages/GroupsPage/SettingsGroupPage.tsx

```diff
@@ -5,25 +5,42 @@ import { useNavigate, useParams } from "react-router-dom";
 import { getErrorMessage } from "api/errors";
 import { group, patchGroup } from "api/queries/groups";
 import { displayError } from "components/GlobalSnackbar/utils";
+import { Loader } from "components/Loader/Loader";
 import { pageTitle } from "utils/page";
 import SettingsGroupPageView from "./SettingsGroupPageView";
 
 export const SettingsGroupPage: FC = () => {
-  const { groupId } = useParams() as { groupId: string };
+  const { groupName } = useParams() as { groupName: string };
   const queryClient = useQueryClient();
-  const groupQuery = useQuery(group(groupId));
+  const groupQuery = useQuery(group(groupName));
   const patchGroupMutation = useMutation(patchGroup(queryClient));
   const navigate = useNavigate();
+  const groupData = groupQuery.data;
+  const isLoading = !groupData;
```

Comment:
not quite! `groupQuery.isLoading` is a better way to check this. `groupQuery.data` can be undefined if the query errors as well. You can check `groupQuery.error` for that as well. It's also really common to show those errors in the UI by doing something like...

```ts
{Boolean(error) && <ErrorAlert error={groupQuery.error} />}
```

Reply:
@aslilac if `groupQuery.isLoading` is better why do we have this on the GroupPage.tsx? `const isLoading = !groupData || !permissions;`

Reply:
because it got missed in code review. it's bad there too, and should be fixed.